### ISSUE TITLE
Version 3.0.0: JSZip not found under Node.js

### ISSUE THREAD

**Issue by rse** [gitbrent/PptxGenJS#638]
When trying to use version 3.0.0 with Node.js it fails on a simple `require("pptxgenjs")` with `ReferenceError: JSZip is not defined` because under Node.js the `main` (from `package.json`) is used and the corresponding `dist/pptxgen.min.js` seems to assume a global(!) `JSZip` symbol (as it is the case just for browser contexts, but not for Node.js contexts). The module` version `dist/pptxgen.es.js` is not used by regular Node.js 12.14. Perhaps you assume a minimum Node.js version of 13.0 where the ESM support exists?

**Comment by afarghaly10** [comment]
I am guessing the issue that in pptxgen.min.js JSZip is not configured. It does not matter which Node version is being used because I am having the same error and upon trying 2 different Node versions on npmjs.com runkit the error still persists 

trying it with Node v13.5
![Screen Shot 2020-01-05 at 1 53 08 AM](https://user-images.githubusercontent.com/33142325/71776362-6f5cb780-2f5e-11ea-8270-a1aaffe57299.png)

with Node v10 and same error shows up 
![Screen Shot 2020-01-05 at 1 41 56 AM](https://user-images.githubusercontent.com/33142325/71776369-91563a00-2f5e-11ea-86c7-75c57f49a0cf.png)

**Comment by gitbrent** [comment]
Hi @rse @afarghaly10 

Thanks for reporting this. I'd been using a local copy for testing and i'm now seeing the same results you guys are after updating the pptxgenjs package to v3.0.0

My misunderstanding of the `module` field in package.json seems to be the cause. Node is using the "main" value even when type:'module' is set.

Using this configuration instead of the current main=min.js enables the node demo to work.

```json
  "browser": "dist/pptxgen.min.js",
  "main": "dist/pptxgen.cjs.js",
```

**Comment by gitbrent** [comment]
Issue has been fixed in v3.0.1

### [Version 3.0.1](https://github.com/gitbrent/PptxGenJS/releases/tag/v3.0.1)

**Comment by gitbrent** [comment]
<img width="761" alt="Screen Shot 2020-01-07 at 23 38 32" src="https://user-images.githubusercontent.com/7218970/71953143-e08eab80-31a6-11ea-923b-f9fef0490421.png">
